### DISCUSSION TITLE
Adding examples of feature flags to show/hide functionality on a website

### DISCUSSION THREAD

**Discussion by clovis1122** [epicweb-dev/epic-stack#432] [Ideas]
I think it would be great to have in the epic-stack as they help greatly with A/B testing and Continuous Deployment. Haven't implemented them in Remix. Can be as easy as checking for a certain parameter in the URL or as complicated as database-backed, random flags. Thoughts?

**Comment by kentcdodds** [comment]
Yep, I definitely want to have something like this. I welcome [an example](https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md).
